activation/post.go

```diff
@@ -89,6 +89,11 @@ func (c *PostClient) Execute(challenge []byte) (*types.PostProof, error) {
 }
 
 func (c *PostClient) Reset() error {
+	ok, err := c.IsInitialized()
+	if !ok || err != nil {
+		return fmt.Errorf("post not initialized, cannot reset it")
```

Comment:
maybe resetting when `err` should work ? (somekind of recovery)

Reply:
no. if an error occurs we should return it, since we do not know what is the status of the file.